daal4py/sklearn/ensemble/_forest.py

```diff
@@ -700,7 +723,9 @@ def __init__(self,
                  verbose=0,
                  warm_start=False,
                  ccp_alpha=0.0,
-                 max_samples=None):
+                 max_samples=None,
+                 maxBins=256,
+                 minBinSize=1):

```

Comment:
How this value was chosen? I see that at least in oneAPI interfaces default `minBinSize = 5`
https://github.com/oneapi-src/oneDAL/blob/master/cpp/oneapi/dal/algo/decision_forest/common.cpp#L60

Reply:
It's for better accuracy for small datasets